Doc/library/pathlib.rst

```diff
@@ -93,6 +93,128 @@ Opening a file::
+:meth:`as_posix() <PurePath.as_posix>`                                                                  String representation of the path with forward slashes (``/``)
+:meth:`is_absolute() <PurePath.is_absolute>`                                                            Whether the path is absolute or not
+:meth:`is_relative_to() <PurePath.is_relative_to>`                                                      Whether the path is relative to the *other* path
+:meth:`is_reserved() <PurePath.is_reserved>`                                                            Whether the path is considered reserved under Windows (deprecated)
+:meth:`joinpath() <PurePath.joinpath>`                                                                  Combine the path with each of the given *pathsegments*
+:meth:`full_match() <PurePath.full_match>`                                                              Match against the provided glob-style pattern
+:meth:`match() <PurePath.match>`                                                                        Match against the provided non-recursive glob-style pattern
+:meth:`relative_to() <PurePath.relative_to>`                                                            Version of this path relative to the *other* path
+:meth:`with_name() <PurePath.with_name>`                                                                New path with the :attr:`PurePath.name` changed
+:meth:`with_stem() <PurePath.with_stem>`                                                                New path with the :attr:`PurePath.stem` changed
+:meth:`with_suffix() <PurePath.with_suffix>`                                                            New path with the :attr:`PurePath.suffix` changed
+:meth:`with_segments() <PurePath.with_segments>`                                                        New path by combining the given *pathsegments*
+
+**Concrete path classes**
+--------------------------------------------------------------------------------------------------------------------------
+:class:`Path() <Path>`                                                                                  Represents concrete paths of the system's path flavour
+:class:`PosixPath() <PosixPath>`                                                                        Represents concrete non-Windows filesystem paths
+:class:`WindowsPath() <WindowsPath>`                                                                    Represents concrete Windows filesystem paths
+
+**Parsing and generating URIs**
+--------------------------------------------------------------------------------------------------------------------------
+:meth:`from_uri() <Path.from_uri>`                                                                      New path from parsing a 'file' URI
+:meth:`as_uri() <Path.as_uri>`                                                                          Represent the path as a 'file' URI
```

Comment:
meth and class roles add parens (according to a sphinx setting), so no need to hard-write them in the source

Reply:
Hmm, I tried removing the parens, and the HTML result didn't show them. Am I doing something wrong?
```
:meth:`as_posix <PurePath.as_posix>` 
:meth:`is_absolute() <PurePath.is_absolute>`        
```
<img width="264" alt="image" src="https://github.com/user-attachments/assets/34fbf74b-1395-4852-8000-e2d4b95f1241" />

Reply:
Oh I see ‑ the thing I mention must only apply by default, but if you specify the label then it’s taken verbatim. Please ignore my note!